sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py

```diff
@@ -3,75 +3,38 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-
-from azure.core.configuration import Configuration
-from azure.core.pipeline import Pipeline
-from azure.core.pipeline.transport import RequestsTransport
-from azure.core.pipeline.policies import (
-    UserAgentPolicy,
-    HeadersPolicy,
-    RequestIdPolicy,
-    ProxyPolicy,
-    NetworkTraceLoggingPolicy,
-    RetryPolicy,
-    RedirectPolicy,
-    BearerTokenCredentialPolicy,
-    DistributedTracingPolicy,
-    HttpLoggingPolicy,
-)
-from ._policies import CognitiveServicesCredentialPolicy, TextAnalyticsResponseHookPolicy
+from ._policies import CognitiveServicesCredentialPolicy
 from ._user_agent import USER_AGENT
 
 
-class TextAnalyticsClientBase(object):
-    def __init__(self, credential, **kwargs):
-        self._pipeline = self._create_pipeline(credential, **kwargs)
-
-    def _create_pipeline(self, credential, **kwargs):
-        credential_policy = None
-        if credential is None:
-            raise ValueError("Parameter 'credential' must not be None.")
-        if hasattr(credential, "get_token"):
-            credential_policy = BearerTokenCredentialPolicy(
-                credential, "https://cognitiveservices.azure.com/.default"
-            )
-        elif hasattr(credential, "api_key"):
-            credential_policy = CognitiveServicesCredentialPolicy(credential)
-        elif credential is not None:
-            raise TypeError("Unsupported credential: {}. Use an instance of TextAnalyticsApiKeyCredential "
-                            "or a token credential from azure.identity".format(type(credential)))
+def _authentication_policy(credential):
+    credential_policy = None
+    if credential is None:
+        raise ValueError("Parameter 'credential' must not be None.")
+    if hasattr(credential, "api_key"):
+        credential_policy = CognitiveServicesCredentialPolicy(credential)
```

Comment:
I thought I saw the core shared credential PR merge. Should we change this to the core policy?

Reply:
i'm leaving this for the other pr, trying to merge the shared credential one first, don't want to muddle the two prs.